src/components/pages/AdminAddCourse/AdminAddCoursesPage.js

```diff
@@ -27,9 +27,11 @@ const initialFormValues = {
   program_id: '',
 };
 
+//until use of Auth0 is in place where we can get a working token
+let authState = null;
+
 function AdminAddCoursesPage(props) {
   const dispatch = useDispatch();
-  const { authState } = useOktaAuth();
   const { idToken } = authState;
```

Comment:
This won't work because `authState` as currently defined is `null` and therefore doesn't have any value associated with `idToken` - let's simply remove or comment out these instances with notes to implement Auth0. 💪